### PR TITLE
[LIDO-4996] move type functionality to ffp

### PR DESCRIPTION
**Pull request by jatkins5**
## Change description

> Description here

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> [LIDO-4996](https://linear.app/trylido/issue/LIDO-4996/move-parsing-overhead-from-client-to-ffp) 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: grammar/hooks.js [hunk 1]
```diff
@@ -381,6 +381,11 @@ class FormulaParser {
      * @returns {*}
      */
     async parseAsyncWithType(inputText, position, allowReturnArray = false) {
+      if (typeof inputText !== 'string' || inputText[0] !== '=') {
+        const res = await this.getResult(inputText);
+        console.log({ res });
```
Comment (tyleradams) [inline grammar/hooks.js]: !

File: grammar/hooks.js [hunk 2]
```diff
@@ -449,6 +459,78 @@ class FormulaParser {
         }
         return res;
     }
+
+    async getResult(text) {
```
Comment (tyleradams) [inline grammar/hooks.js:461]: Strip out =

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lido-app/fast-formula-parser",
-  "version": "7.0.0",
+  "version": "7.1.0",
```
Comment (tyleradams) [inline package.json]: 8.0.0

**Review by tyleradams — APPROVED** [review]

**Comment by max-prtsr** [comment]
image


Guys, these commits must be squashed before merge.

**Pull request merged**